### PR TITLE
Update test_acronym.c

### PR DESCRIPTION
**Pull request by kgashok**
added all lower case test

### CODE REVIEW

File: exercises/acronym/test/test_acronym.c [hunk 1]
```diff
@@ -74,6 +74,14 @@ void test_non_acronym_all_caps_words(void)
    test_abbreviation(phrase, expected);
 }
 
+void test_all_lower_case(void)
```
[the remark this answers is not in the document]
Comment (kgashok) [reply exercises/acronym/test/test_acronym.c]: Done!

[the remark this answers is not in the document]
Comment (kgashok) [reply exercises/acronym/test/test_acronym.c]: If I do an `exercism restore`, will the test cases for this exercise get updated?

**Comment by wolf99** [comment]
Does the "Ruby on Rails" case not cover lowercase words?

If not, maybe we could locate the non-problem-specification test cases together instead of scattered through the file, for ease of parsing by future readers. Currently the non-problem-specification cases are the NULL case, "PHP" and this "FWIW" case.

**Comment by kgashok** [comment]
"Ruby on Rails" does not start with a lowercase. "FWIW" is a boundary case.

**Comment by wolf99** [comment]
LGTM 👍

**Comment by ryanplusplus** [comment]
@kgashok the list of problems comes from https://github.com/exercism/problem-specifications. @wolf99 was suggesting to keep all of the problems from the upstream problem-specifications repo together.

**Pull request merged**